cpp/include/ray/api/ray_config.h

```diff
@@ -9,45 +9,31 @@
 namespace ray {
 namespace api {
 
-enum class RunMode { SINGLE_PROCESS, CLUSTER };
-
-/// TODO(Guyang Song): Make configuration complete and use to initialize.
 class RayConfig {
  public:
-  WorkerType worker_type = WorkerType::DRIVER;
-
-  RunMode run_mode = RunMode::SINGLE_PROCESS;
-
-  std::string redis_ip;
-
-  int redis_port = 6379;
-
-  std::string redis_password = "5241590000000000";
+  // The redis address of existing cluster.
```

Comment:
```suggestion
  // The address of the Ray cluster to connect to.
```

I copied from `worker.py`.